cms/templates/maintenance/_announcement_edit.html

```diff
@@ -0,0 +1,47 @@
+<%page expression_filter="h"/>
+<%inherit file="base.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%block name="title">${_('Edit Announcement')}</%block>
+<%block name="viewtitle">
+<h3 class="info-course">
+    <span>${_('Edit Announcement')}</span>
+</h3>
+</%block>
+
+<%block name="viewcontent">
+<section class="container maintenance-content">
+    <div class="container studio-view maintenance-form">
+        <form class="form-create" method="post" action="${action_url}">
+            <div class="wrapper-form announcement-container">
+                <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+                <div class="announcement-item">
+                  ${form.as_p() | n} ## xss-lint: disable=mako-invalid-html-filter
+                </div>
+                <div class="actions">
+                    <button type="submit" class="action action-primary">${_('Save')}</button>
+                </div>
+            </div>
+        </form>
+    </div>
+</section>
+</%block>
+
+<%block name="header_extras">
+  <script src="${STATIC_URL}js/vendor/tinymce/js/tinymce/tinymce.full.min.js"></script>
+  <script type="text/javascript">
+      tinymce.init({
+          selector: "#id_content",
+          branding: false,
+          plugins: [
+              "advlist autolink lists link image charmap print preview anchor",
+              "searchreplace visualblocks code fullscreen",
+              "insertdatetime media table contextmenu paste"
+          ],
```

Comment:
Since `lists` don't render very nicely or consistently between the CMS, LMS, and the `preview` and `fullscreen` views are also misleading, maybe it's better to just remove all of these options?

This would also reduce the number of tinymce plugins added.

* CMS:
  ![cms lists](https://user-images.githubusercontent.com/7556571/51825785-72eec400-2335-11e9-9733-136c9ac5cad2.png)
* LMS:
  ![lms lists](https://user-images.githubusercontent.com/7556571/51825793-76824b00-2335-11e9-9c20-e704abc2ccf2.png)
* Preview: 
  ![preview](https://user-images.githubusercontent.com/7556571/51826016-f5778380-2335-11e9-917b-4c0f1b52ad25.png)
* Fullscreen: 
  ![fullscreen](https://user-images.githubusercontent.com/7556571/51826024-fad4ce00-2335-11e9-8c04-e61af304ff0c.png)
  Also, I couldn't find a way to get back out of Fullscreen?

Reply:
Quite right, I've removed those plugins as they aren't really essential.
Those were only included because I copied the iist of plugins that were in use before.
I've pruned them now, and regenerated the TinyMCE plugins so they're not included.